### ISSUE TITLE
Set the default value for "Keep events"?

### ISSUE THREAD

**Issue by hunhejj** [cantino/huginn#1724]
Is it possible to set the default value for `Keep events` that most agent have to a user-defined value? By default events will be kept forever, i.e. every time I create a new agent I have to reset this option to a limited a value since I don't want to keep the events for too long.

Btw. is there a description somewhere what options can be set in the env file?

**Comment by cantino** [comment]
No, this isn't currently possible.  You could contribute this feature if you're interested in working on it.

The the `.env.example` file should have all possible options.

**Comment by hunhejj** [comment]
I will check if this is something I could solve. 

I totally forgot about `env.example`, that's exactly what I was looking for!
